src/Appwrite/Platform/Workers/Builds.php

```diff
@@ -542,9 +542,11 @@ protected function buildDeployment(Device $deviceForFunctions, Func $queueForFun
                 ->addMetric(METRIC_BUILDS, 1) // per project
                 ->addMetric(METRIC_BUILDS_STORAGE, $build->getAttribute('size', 0))
                 ->addMetric(METRIC_BUILDS_COMPUTE, (int)$build->getAttribute('duration', 0) * 1000)
+                ->addMetric(METRIC_BUILDS_MB_SECONDS, $function->getAttribute('memory') * $build->getAttribute('duration', 0))
```

Comment:
Is `duration` seconds or millis? I see some places we multiply by 1000

Reply:
It is stored in seconds: https://github.com/appwrite/appwrite/blob/cbea4264eaf72eb8bde43a2cd8eb24103904ad0c/src/Appwrite/Utopia/Response/Model/Build.php#L60-L65